### DISCUSSION TITLE
facing errors after updating

### DISCUSSION THREAD

**Discussion by Buravo** [timeseriesAI/tsai#628] [Q&A]
Hi,
After yesterdays updates(I guess), some methods that I previously successfully utilized (1. learn.plot_top_losses 2.learn.top_losses 3. learn.feature_importance) all of sudden haven't worked anymore with errorname: 
> TypeError: unhashable type: 'numpy.ndarray' 

Could you please help me solving with that-?
I've tried to use previous version (0.3.2) but couldn't solve it..

**Comment by oguiza** [comment]
Hi @Buravo ,
Could you please provide a code snippet to reproduce the issue?
I've tested it now and it seems to work well (see [gist](https://colab.research.google.com/gist/oguiza/b7d0fb9b815904e4022d25ee5080f6a5/scratchpad.ipynb)).

**Reply by Buravo** [reply]
@oguiza , Truly appreciate for the swift reply ! I've checked the gist codes you provided and reviewed it works well. However, the error I face still happens when using sliding window panel. Will send you detail via email shortly. Thank you very much for the awesome work, btw 👍
